### ISSUE TITLE
Changing rate limit in aws_wafregional_rate_based_rule does not result in an update

### ISSUE THREAD

**Issue by splieth** [terraform-providers/terraform-provider-aws#5355]
When I want to update the ```rate_limit``` in a ```aws_wafregional_rate_based_rule``` resource, terraform plan actually says that there will be an update and that the resource changed. Unfortunately, the update is not performed due to the implementation of the resource. Therefore, the limit remains the same.

### Terraform Version

```Terraform v0.11.7```

### Affected Resource(s)

* aws_wafregional_rate_based_rule

### Terraform Configuration Files

```hcl
resource "aws_wafregional_rate_based_rule" "some_rule" {
  name = "some_metric"
  metric_name = "some_metric"
  rate_key = "IP"
  rate_limit = 2000
}
```

### Expected Behavior

Rate limit should have been updated.

### Actual Behavior

Rate limit is not updated.

### Steps to Reproduce

1. `terraform apply`
2. Change rate limit:
```hcl
resource "aws_wafregional_rate_based_rule" "some_rule" {
  name = "some_metric"
  metric_name = "some_metric"
  rate_key = "IP"
  rate_limit = 2001
}
```
3. `terraform apply`

**Comment by bflad** [comment]
The fix for this has been merged into master and will release with version 1.30.0 of the AWS provider

**Comment by bflad** [comment]
This has been released in [version 1.30.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1300-august-02-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Comment by peterschmidt85** [comment]
I experienced the same issue with the 2.x version of the AWS provider. Anyone else?
